modules/core/map/modules/google/farm_map_google.links.task.yml

```diff
@@ -0,0 +1,4 @@
+farm_map_google.settings:
+  title: 'Google Maps'
```

Comment:
I wonder if we should change this to simply `title: 'Google'`, so the three tabs that are visible under `/farm/settings/map/` are: Map / Google / Mapbox